Twitch/Twitch.Base/Clients/ChatClient.cs

```diff
@@ -90,6 +90,12 @@ private enum MessageState
         /// </summary>
         public event EventHandler<ChatMessagePacketModel> OnMessageReceived;
 
+        /// <summary>
+        /// Invoked when a whisper is received.
+        /// </summary>
+        public event EventHandler<ChatWhisperMessagePacketModel> OnWhisperMessageReceived;
```

Comment:
Just for awareness, PubSub already has alerts on whispers, but doesn't hurt to also have it here.

Reply:
The issue with PubSub is that a bot running under only a moderator level account can't subscribe to the events.  PubSub requires the owner's token.  This will allow it to work in those situations.

Reply:
Makes sense, just wanted to let you know if you hadn't seen it.